test/powershell/Modules/Microsoft.PowerShell.Core/PSSessionConfiguration.Tests.ps1

```diff
@@ -497,6 +501,15 @@ namespace PowershellTestConfigNamespace
                     $Result.Session.UseSharedProcess | Should be $UseSharedProcess
                 }
 
+                It "Validate Register-PSSessionConfiguration -PSVersion" {
+
+                Register-PSSessionConfiguration -Name $TestSessionConfigName -PSVersion 5.1
+                $Session = Get-PSSessionConfiguration -Name $TestSessionConfigName
+
+                $Session.Name | Should be $TestSessionConfigName
+                $Session.PSVersion | Should Be 5.1
+            }
```

Comment:
Please fix the indentation of this it block.

Reply:
Fixed.